src/Microsoft.ML.StaticPipe/ModelOperationsCatalog.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace Microsoft.ML.StaticPipe
+{
+    public static class ModelOperationsCatalog
+    {
+        /// <summary>
+        /// Save statically typed model to the stream.
+        /// </summary>
+        /// <param name="catalog">The model explainability operations catalog.</param>
+        /// <param name="model">The trained model to be saved.</param>
+        /// <param name="stream">A writeable, seekable stream to save to.</param>
+        public static void Save<TInShape, TOutShape, TTransformer>(this ML.ModelOperationsCatalog catalog, Transformer<TInShape, TOutShape, TTransformer> model, Stream stream)
```

Comment:
>ModelOperationsCatalog catalog [](start = 75, length = 30)

This is an interesting question I'm not sure I know the answer to. The experimental "static" API predates the concept of the `MLContext`, and solved the problem `MLContext` seeks to solve -- that of discoverability, and (at a lower level) propagation of the `IHostEnvironment`, in a rather entirely different way -- that is, you apply estimators and transformers in static-pipe land through discoverability on the input data to those estimators and transformers -- that is, intellisense on the columns.

Dynamic pipe land can't do that -- there is no (and kind of cannot be) concept of a column as "separate object" in dynamic-pipe land. So we achieve discoverability of relevant instantiable components via the taxonomy as originally proposed in #1098.

Here we're, AFAIK for the first time, mixing the two metaphors. This may be appropriate. Indeed I think we might shift the static API to use more of these idioms (which, again, predates `MLContext` altogether during the period we were engineering its core principles).

Not asking you to answer that question, just registering my thoughts since this change brought it to my mind, if we continue to iterate on the static pipe...